cloud/aws/templates/aws_oidc/variables.tf

```diff
@@ -507,3 +507,14 @@ variable "postgresql_major_version" {
   description = "Major version of PostgreSQL to use"
   default     = 16
 }
+
+variable "external_vpc" {
+  type        = map(string)
+  description = "A map with external VPC settings"
```

Comment:
**suggestion:** It may be worth noting that all fields in the map must be set for the external VPC to be used.